CometServer/Modules/Constraints/EnumerableOfGuidRouteConstraint.cs

```diff
@@ -0,0 +1,63 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="EngineeringModelApi.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2024 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Antoine Théate
+//
+//    This file is part of CDP4-COMET Webservices Community Edition. 
+//    The CDP4-COMET Webservices Community Edition is the RHEA implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+//
+//    The CDP4-COMET Webservices Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+//
+//    The CDP4-COMET Webservices Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CometServer.Modules.Constraints
+{
+    using System;
+
+    using CometServer.Extensions;
+
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Routing;
+
+    /// <summary>
+    /// The purpose of the <see cref="EnumerableOfGuidRouteConstraint" /> is to enable url routing for GUID
```

Comment:
should this not read:

```
The purpose of the <see cref="EnumerableOfGuidRouteConstraint" /> is to enable url routing for enumerable of shortguid?
```

not sure whether shortguid is the right term, but what is important to me is that is states that it is more than 1 guid

Reply:
Agreed, updated to 
![image](https://github.com/RHEAGROUP/COMET-WebServices-Community-Edition/assets/61700723/18ad63f1-d6af-41b8-9098-2bbae1934e33)